util/aggregation.py

```diff
@@ -358,6 +352,16 @@ def aggregate_csv(job_dir, out_dir, request_id=None):
                     os.path.join(out_dir, f))
 
 
+@log_timing(log)
+def aggregate_json(job_dir, out_dir, request_id=None):
+    # TODO -- aggregate JSON files
```

Comment:
Is this still work needing to be done? If not, the comment should be removed.

Reply:
Yes, work is still needed. The aggregate_json code here just copies the files from the sub-job directories to the async output directory (e.g. from /local/async_results/<user>/<subjob dir> to /opt/ooi/async/<user>/<async output dir>). It does not try to combine the files in any way. Without the changes above, the particle data files don't even appear in the async output directory. I figured JSON and CSV aggregation should be fixed to actually combine files at the same time and as part of a separate effort (it seems low priority since I haven't heard complaints about CSV and JSON async output).

Reply:
can the comment be clarified to indicate that the TODO is to combine files based on time?  right now it's kind of confusing comment because the method is called aggregate_json and the comment says TODO aggregate JSON files.